[FEATURE] Redirect root to host meta, link to docs

stevenzeiler opened the pull request:
[FOLD] Clean up require module routes

wilsonianb commented:
Can you add this to the commit message:
`[Finishes #89446660]`
`[Finishes #89445636]`

wilsonianb commented on lib/application.js:
```diff
@@ -1,17 +1,18 @@
-var path     = require('path');
-var express  = require('express');
+var path     = require('path'); var express  = require('express');
```
Two requires on one line

wilsonianb commented on routes/host_meta.js:
```diff
@@ -0,0 +1,7 @@
+module.exports = function(req, res) {
+  res.status(200).send({
+    properties: {
+      documentation: "https://codius.org/docs/using-codius/getting-started"
```
Should this be: https://codius.org/docs/running-a-host/setup

Commit:
[FEATURE] Host meta endpoint

[FEATURE] Redirect root to host meta, link to docs
[FEATURE] List codius host version in host meta
[Finishes #89446660]
[Finishes #89445636]

[FOLD] Requires on separate lines
